examples/000_Local_Deployment.ipynb

```diff
@@ -0,0 +1,352 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"GET /user/list-users HTTP/1.1\" 200 26\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['default_user@localhost']"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "auth_backend_call('get', '/user/list-users', token=access_token).json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now lets add a user"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"POST /user/create-user HTTP/1.1\" 201 25\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'detail': 'User created'}"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "payload = payload={\n",
+    "  \"email\": \"user2@email.com\",\n",
+    "  \"password\": \"test_password\"\n",
+    "}\n",
+    "\n",
+    "auth_backend_call('post', '/user/create-user', payload=payload, token=access_token).json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now when we list the users, we see two users. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"GET /user/list-users HTTP/1.1\" 200 44\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['default_user@localhost', 'user2@email.com']"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "auth_backend_call('get', '/user/list-users', token=access_token).json()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "dtbase_env",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
Optional, but we could also add here a section we screenshots for how to do this using the frontend.